tests/test_percy_snapshot.py

```diff
@@ -74,36 +68,20 @@ def display_image(pathname):  # pylint: disable=W0612
 
         return app
 
-    def percy_snapshot(self, name=''):
+    @staticmethod
+    def percy_snapshot(dash_duo, name=''):
         if os.environ.get('PERCY_ENABLED', False):
```

Comment:
You don't need to explicitly check `PERCY_ENABLED` - `dash_duo.percy_snapshot` is a noop if percy is disabled.